graviti/portex/ptype.py

```diff
@@ -286,7 +294,9 @@ def load(content: Dict[str, TypingAny], imports: Optional[Imports] = None) -> Cl
             A Portex type instance created from the input python dict.
 
         """
-        return ClassPortexType.from_pyobj(content, imports)
+        if content is not None:
+            return ClassPortexType.from_pyobj(content, imports)
+        return content
```

Comment:
under what condition the portex type will be null?

Reply:
`visible` in keypoints.